src/CommunityToolkit/Xamarin.CommunityToolkit/Helpers/LocalizationResourceManager.shared.cs

```diff
@@ -30,10 +30,10 @@ public void Init(ResourceManager resource, CultureInfo initialCulture)
 			CurrentCulture = initialCulture;
 		}
 
-		public string GetValue(string text) =>
-			resourceManager.GetString(text, CurrentCulture);
+		public string? GetValue(string text) =>
```

Comment:
I believe that here we want the resourManager to throw a NRE, is a way to tell the dev that he needs to call the Init method.